playgrounds/vite/src/main.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+import styled from '@emotion/styled';
+import { useTheme } from '@itwin/itwinui-react';
+import App from './App';
+import { css, Global } from '@emotion/react';
+
+const Shell = () => {
+  const [theme, setTheme] = React.useState<'light' | 'dark'>(() =>
+    matchMedia('(prefers-color-scheme: dark)').matches ? 'dark' : 'light',
+  );
+
+  useTheme(theme);
+
+  return (
+    <>
+      <Global
+        styles={css`
+          * {
+            box-sizing: border-box;
+            margin: 0;
+          }
+        `}
+      />
```

Comment:
I think this is an addition to the old `itwinui-playground`. Just curious, what does this do and why was this needed?

Reply:
this is a replacement for the `main.css` file which i deleted. i'm trying to keep this repo free of css files.